.github/workflows/test.yml

```diff
@@ -23,10 +23,12 @@ jobs:
         - '3.8'
         - '3.9'
         - '3.10'
+        - '3.11'
         django-version:
         - '2.2.*'
```

Comment:
@philfriesen are you still using `2.2.x`? Or can this be removed? 

[2.2 has been out of date as of 7 months ago.](https://endoflife.date/django)

Reply:
@kingbuzzman We are on 3.9 now. Thanks!

Reply:
Yes, fine with us.